catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -243,48 +243,60 @@ public ReliableResourceStatus call() {
 
                 bytesRead.addAndGet(n);
 
-                if (cacheFileOutputStream != null) {
```

Comment:
Actually @lessarderic now that I reflect on it more, it really makes sense to scrap this refactoring for a separate ticket. This threading stuff will take some thought to get right and there's no point to holding up the main focus.